### PR TITLE
Add screen logging information to the support logs

### PR DESCRIPTION
**Pull request by radujipa**
This attaches a screen-log.txt file to the existing logs with
information from kano-settings display functions.

### CODE REVIEW

**Comment by radujipa** [comment]
[(2) As a customer care agent, I want users to send us the screen logs](https://trello.com/c/2OelJC9i/1738-2-as-a-customer-care-agent-i-want-users-to-send-us-the-screen-logs)

File: kano_feedback/DataSender.py [hunk 1]
```diff
@@ -372,6 +372,43 @@ def get_hdmi_info():
...
+        log_data = {
+            'model': model,
+            'status': status,
+            'edid': edid,
+            'supported': supported,
+            'optimal': optimal,
+        }
+        log = json.dumps(log_data, sort_keys=True, indent=4)
+    except:
+        import traceback
```
Comment (skarbat) [inline kano_feedback/DataSender.py]: Only a suggestion, importing stuff in an exception looks a bit risky to me.

Comment (radujipa) [reply kano_feedback/DataSender.py]: Good point! It was merely to avoid importing a bunch of things unless required. Fixed 👍

Comment (wizofe) [reply kano_feedback/DataSender.py]: That's a good point. You may want to `try` to import inside the `exception` (although maybe nesting multiple exceptions isn't what you wanted)!

```
except:
    try:
        import traceback
    except ImportError:
         foo
```

File: kano_feedback/DataSender.py [hunk 2]
```diff
@@ -372,6 +373,40 @@ def get_hdmi_info():
...
+        log_data = {
+            'model': model,
+            'status': status,
+            'edid': edid,
+            'supported': supported,
+            'optimal': optimal,
+        }
+        log = json.dumps(log_data, sort_keys=True, indent=4)
+    except:
+        return traceback.format_exc()
```
Comment (Ealdwulf) [inline kano_feedback/DataSender.py]: Nice.

**Pull request merged**